imu_transformer/include/imu_transformer/tf2_sensor_msgs.h

```diff
@@ -32,7 +32,7 @@
 #ifndef TF2_SENSOR_MSGS_H
 #define TF2_SENSOR_MSGS_H
 
-#include <tf2/convert.h>
+#include <tf2/convert.hpp>
```

Comment:
we should really wrap this with an if/else so that the branch continues to compile on older systems (since I expect this change to tf2 will only be released into rolling and later at this point):

```cpp
#if __has_include(<tf2/convert.hpp>)
#include <tf2/convert.hpp>
#else
#include <tf2/convert.h>
#endif

Reply:
This PR was meant to be preemptive one, that being said I'll probably do some backwards compatibility in which earlier branches like Humble will have the .h files include the .hpp files without a warning to prevent this from happening. In that case syntax like this shouldn't be needed.